File: javascript-solutions/hackerrank/algorithms/veryBigSum.js

function aVeryBigSum(ar) {
    return ar.reduce( (acc,n) => acc + n );
}

function main() {
    const arr = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]
    const sum = aVeryBigSum(arr)
    console.log( sum )
    return sum
}

File: rust-solutions/hackerrank/algorithms/src/bin/very-big-sum.rs

fn a_very_big_sum(arr: &[i64]) -> i64 {
    arr.iter().sum()
}

fn main() {
    let arr = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005];
    println!("{}", a_very_big_sum(&arr))
}